### PR TITLE
Added CSV framework and an article on vapor 3.

### PR DESCRIPTION
**Pull request by proggeramlug**
- **Project Name**: CSV Framework
- **Project URL**: https://github.com/skelpo/CSV
- **Project Description**: A simple CSV parser written specifically for and with Vapor.
- **Why it should be added to `awesome-vapor`**:
- [ ] Project has at least 15 stars (if it's a GitHub project)
- [x] Project supports Swift 4
- [x] Project supports at least Vapor 2
- [ ] I used correctly sized dash and ended the entry with a full stop 🤓

### CODE REVIEW

**Comment by Voilane** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          2 Errors
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please make sure your submission uses correctly sized dash and ends with a full stop</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 2 link issues</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
71 | [404](https://httpstatuses.com/404) | https://bygri.github.io/2018/01/25/vapor-2-with-docker.html 
72 | [404](https://httpstatuses.com/404) | https://bygri.github.io/2018/01/25/vapor-3-with-docker.html 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by Cellane** [comment]
I fixed the two link issues the bot was complaining about in PR #18 so you can resubmit if you wish – or I can do so myself a bit later :)

**Comment by proggeramlug** [comment]
@Cellane thanks! I was a little confused. I accidentally added two links with one PR, next time I'll do it better ;)

**Comment by Cellane** [comment]
@proggeramlug It's totally fine, most of the warnings are just suggestions for now to see what sticks, what makes sense and what actually works for the repository in the long run. I wouldn't want to pester people endlessly (perhaps except when it's typography related 🤣) and chase everyone away 😅 

Thanks for your contribution, at any rate!

File: README.md [hunk 1]
```diff
@@ -67,8 +68,9 @@ Simply press <kbd>Command</kbd> + <kbd>F</kbd> to search for a keyword.
 - ![v2](img/vapor-2.png) [Fluent Part 4 - Persistence With MySQL And Fluent Patterns](https://geeks.brokenhands.io/blog/posts/fluent-part-4-persistence-with-mysql-and-fluent-patterns/)
 - ![v2](img/vapor-2.png) [Getting started with Vapor](https://geeks.brokenhands.io/blog/posts/getting-started-with-vapor/)
 - ![v2](img/vapor-2.png) [Leaf Part 1 - Getting Started](https://geeks.brokenhands.io/blog/posts/leaf-part-1-getting-started/)
-- ![v2](img/vapor-2.png) [Vapor 2 with Docker](https://bygri.github.io/2018/01/24/vapor-2-with-docker.html) – Definitive Docker guide for Vapor 2.
-- ![v3](img/vapor-3.png) [Vapor 3 with Docker](https://bygri.github.io/2018/01/24/vapor-3-with-docker.html) – Definitive Docker guide for Vapor 3.
+- ![v2](img/vapor-2.png) [Vapor 2 with Docker](https://bygri.github.io/2018/01/25/vapor-2-with-docker.html) – Definitive Docker guide for Vapor 2.
+- ![v3](img/vapor-3.png) [Vapor 3 with Docker](https://bygri.github.io/2018/01/25/vapor-3-with-docker.html) – Definitive Docker guide for Vapor 3.
```
[the remark this answers is not in the document]
Comment (proggeramlug) [reply README.md]: Well, I honestly just did all of this through the web interface of GitHub - should probably check it out locally.

[the remark this answers is not in the document]
Comment (Cellane) [reply README.md]: Ah, I see! No, it's fine, let me find a way to merge it despite “failing” “checks” and if it breaks the links, I'll just fix that later :)

**Comment by Cellane** [comment]
Merged, thank you for your contribution! 🙇

**Pull request merged**